common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
@lamagnifica Is it necessary to include the "edX" qualifier for the mobile app?  Would it suffice if the 2nd sentence were:  "If you provide a URL, the mobile app includes a button that students can tap to access the group."
This would address @aleffert's point about keeping it platform-agnostic.